anomalib/models/__init__.py

```diff
@@ -95,7 +94,8 @@ def get_model(config: Union[DictConfig, ListConfig]) -> AnomalyModule:
     else:
         raise ValueError(f"Unknown model {config.model.name}!")
 
-    if "init_weights" in config.keys() and config.init_weights:
-        model.load_state_dict(load(os.path.join(config.project.path, config.init_weights))["state_dict"], strict=False)
+    # This is needed for nncf. Might possibly break nncf but are tests don't cover nncf.
+    if "ckpt_path" in config.keys() and config.ckpt_path:
+        model.load_state_dict(load(config.ckpt_path)["state_dict"], strict=False)
```

Comment:
Is this really needed? This kind of beats the purpose of using PL's checkpoint loading mechanism

Reply:
I think this has something to do with nncf but since we don't test that we can remove it. Plus I can see in the tests that the nncf tests set this to None.